crates/notion-core/src/path/windows.rs

```diff
@@ -164,12 +170,16 @@ pub fn shim_file(toolname: &str) -> Fallible<PathBuf> {
 //                         catalog.toml                user_catalog_file
 
 fn local_data_root() -> Fallible<PathBuf> {
-    #[cfg(windows)]
-    return Ok(winfolder::Folder::LocalAppData.path().join("Notion"));
+    if let Some(home_dir) = env::home_dir() {
```

Comment:
I'm using this to sandbox the test environment, since otherwise the `$HOME\AppData\Local\` folder will be accessed directly, under the user that is running the tests instead of the home directory in the sandbox.

Reply:
We talked on the phone this morning and came up with a strategy. Here's a synopsis:

- We'll make a `NOTION_SANDBOX` environment variable to make it clearer when our paths are being computed for a sandboxed root directory. Our acceptance tests are not testing the standard directories, but that's true no matter what; this just makes the logic easier to follow and maintain.
- In a separate PR, we'll make one or two smoke tests that require a global testing lock, clear out the Notion directories and reset them to a default state, and just test a few basic workflows. (We should file an issue for this.)
- I'll add some unit tests to the `winfolder` crate.

Reply:
Added an issue for smoke tests: https://github.com/notion-cli/notion/issues/162